docs/mocks.md

```diff
@@ -0,0 +1,115 @@
+---
+id: mocks
+title: Test Mocking
+---
+
+The mocking mechanism enables recording and playback of traffic to a file, instead of sending traffic to the integration instance.
+
+The mocking mechanism is not used for tests that have no integrations.
+
+## Configuration
+Currently, the only configuration option for the mocking mechanism is to exclude integrations from it.
+You can do this by adding the **integration id** and a reason to the **unmockable_integrations** dictionary in **Tests/conf.json**
+
+## Mocking mechanism outputs in CircleCI
```

Comment:
this document should be in internal content , shouldn't be public in my opinion. Few customers and Francesco got confused between this mocks and unit test mocks and what are we talking here

Reply:
Will remove. We have it in internal already.